calendar/calendar-summary-tool/tool/src/java/org/sakaiproject/tool/summarycalendar/ui/CalendarBean.java

```diff
@@ -862,22 +862,26 @@ private String buildEventUrl(Site site, String eventRef) {
 	public synchronized Map<String, String> getEventIconMap() {
 		if(eventIconMap == null || eventIconMap.size() == 0){
 			eventIconMap = new HashMap<String, String>();
-			eventIconMap.put("Academic Calendar", "<span class=\"icon icon-calendar-academic-calendar\"></span>");
-			eventIconMap.put("Activity", "<span class=\"icon icon-calendar-activity\"></span>");
-			eventIconMap.put("Cancellation", "<span class=\"icon icon-calendar-cancellation\"></span>");
-			eventIconMap.put("Class section - Discussion", "<span class=\"icon icon-calendar-class-section-discussion\"></span>");
-			eventIconMap.put("Class section - Lab", "<span class=\"icon icon-calendar-class-section-lab\"></span>");
-			eventIconMap.put("Class section - Lecture", "<span class=\"icon icon-calendar-class-section-lecture\"></span>");
-			eventIconMap.put("Class section - Small Group", "<span class=\"icon icon-calendar-class-section-small-group\"></span>");
-			eventIconMap.put("Class session", "<span class=\"icon icon-calendar-class-session\"></span>");
-			eventIconMap.put("Computer Session", "<span class=\"icon icon-calendar-computer-session\"></span>");
-			eventIconMap.put("Deadline", "<span class=\"icon icon-calendar-deadline\"></span>");
-			eventIconMap.put("Exam", "<span class=\"icon icon-calendar-exam\"></span>");
-			eventIconMap.put("Meeting", "<span class=\"icon icon-calendar-meeting\"></span>");
-			eventIconMap.put("Multidisciplinary Conference", "<span class=\"icon icon-calendar-multidisciplinary-conference\"></span>");
-			eventIconMap.put("Quiz", "<span class=\"icon icon-calendar-quiz\"></span>");
-			eventIconMap.put("Special event", "<span class=\"icon icon-calendar-special-event\"></span>");
-			eventIconMap.put("Web Assignment", "<span class=\"icon icon-calendar-web-assignment\"></span>");
+			eventIconMap.put("Academic Calendar", "<i class=\"icon icon-calendar-academic-calendar\"></i>");
```

Comment:
Good spot, the italic got swapped with a span in a previous Jira but this cherry-pick from Ox code didn't reflect this.  I'll change them and re-commit.